pom.xml

```diff
@@ -3084,6 +3084,9 @@
         <hive.shims.scope>provided</hive.shims.scope>
         <orc.classifier>nohive</orc.classifier>
         <datanucleus-core.version>3.2.10</datanucleus-core.version>
+        <hadoop.version>2.7.4</hadoop.version>
+        <curator.version>2.7.1</curator.version>
+        <commons-io.version>2.4</commons-io.version>
```

Comment:
Does this mean the builds now pass even with `-Phadoop-3.2 -Phive-1.2` but setting the Hadoop version to 2.7 implicitly?

Reply:
Ya. This looks completely wrong.